IAteIt/IAteIt/View/MyProfile/Component/ProfileCellView.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ProfileCellView.swift
+//  IAteIt
+//
+//  Created by Youngwoong Choi on 2023/04/08.
+//
+
+import SwiftUI
+
+struct ProfileCellView: View {
+    
+    let paddingLR: CGFloat = 16
+    let profileImgSize: CGFloat = 120
+    
+    var body: some View {
+        
+        VStack {
+            ZStack {
+                Circle()
+                    .frame(width: profileImgSize, height: profileImgSize)
+                Image("Sample_Profile1")
+                    .resizable()
+                    .scaledToFill()
+                    .layoutPriority(-1)
+            }
+            .clipped()
+            .cornerRadius(profileImgSize/2)
+            
+            Text("\(User.user1.nickname)")
```

Comment:
```suggestion
            Text("\(User.users[0].nickname)")
```
빌드하면 이부분에서 에러가 나는데, 위와 같이 수정하는 것이 맞을까요..?

Reply:
네 수정해서 다시 커밋할게요!